scripts/flake_checker.py

```diff
@@ -29,7 +32,7 @@
     '/report-pass')
 REPORT_API_KEY = '7Ccp062JVjv9LUYwnLMqcm5Eu5gYqqhpl3zQmcO3cDQ'
 
-CI_INFO = {
+CI_INFO: Dict[str, CommonCIInfoDict] = {
```

Comment:
I think we can't use `Final` here because that will give errors like this
### Value of type "Collection[str]" is not indexable  [index]
and for that, we need to use `TypedDict`

Reply:
Hmm. When we use `Final`, when I tried revealing `info['env']`, this is what I found `Revealed type is 'typing.Collection*[builtins.str]'` rather than a `Dict`. Not sure why.